frontend/javascripts/oxalis/model/sagas/update_actions.ts

```diff
@@ -183,30 +209,40 @@ export type UpdateAction =
   | DeleteEdgeUpdateAction
   | UpdateSkeletonTracingUpdateAction
   | UpdateVolumeTracingUpdateAction
-  | UpdateUserBoundingBoxesAction
-  | CreateSegmentVolumeAction
-  | UpdateSegmentVolumeAction
-  | DeleteSegmentVolumeAction
+  | UpdateUserBoundingBoxesUpdateAction
+  | CreateSegmentUpdateAction
+  | UpdateSegmentUpdateAction
+  | DeleteSegmentUpdateAction
   | UpdateBucketUpdateAction
   | UpdateTreeVisibilityUpdateAction
   | UpdateTreeGroupVisibilityUpdateAction
   | RevertToVersionUpdateAction
   | UpdateTreeGroupsUpdateAction
-  | RemoveFallbackLayerAction
-  | UpdateTdCameraAction;
+  | RemoveFallbackLayerUpdateAction
+  | UpdateTdCameraUpdateAction
+  | UpdateMappingNameUpdateAction
+  | SplitAgglomerateUpdateAction
+  | MergeAgglomerateUpdateAction;
 // This update action is only created in the frontend for display purposes
 type CreateTracingUpdateAction = {
   name: "createTracing";
   value: {};
 };
+// This update action is only created by the backend
+type ImportVolumeTracingUpdateAction = {
+  name: "importVolumeTracing";
+  value: {
+    largestSegmentId: number;
+  };
+};
 type AddServerValuesFn<T extends { value: any }> = (arg0: T) => T & {
   value: T["value"] & {
     actionTimestamp: number;
   };
 };
 
 type AsServerAction<A extends { value: any }> = ReturnType<AddServerValuesFn<A>>;
-// Since flow does not provide ways to perform type transformations on the
+// Since typescript does not provide ways to perform type transformations on the
```

Comment:
I was curious and checked whether TS might be smarter than flow and in fact it is :tada: The entire ServerUpdateAction can be slimmed down to:

```ts
export type ServerUpdateAction = AsServerAction<
  UpdateAction | ImportVolumeTracingUpdateAction | CreateTracingUpdateAction
>;
```

Reply:
That's awesome! Thanks for testing!